metadata-ingestion/src/datahub/utilities/sqlglot_lineage.py

```diff
@@ -588,6 +588,9 @@ def _schema_aware_fuzzy_column_resolve(
     except sqlglot.errors.OptimizeError as e:
         # This is not a fatal error, so we can continue.
         logger.debug("sqlglot failed to annotate types: %s", e)
+    except sqlglot.errors.ParseError as e:
+        # This is not a fatal error, so we can continue.
+        logger.debug("sqlglot failed to parse types: %s", e)
```

Comment:
Let's merge this into the previous statement - `except` for both exception types

Reply:
done